README.md

```diff
@@ -1,7 +1,132 @@
-Kotlin-template
-============
+TickTock
+========
 
-Template project for Kotlin projects.
+TickTock is a timezone data management library for the JVM and Android targeting `java.time.*` APIs 
+in Java 8 or above. Use this library if you want to bundle timezone data directly with your 
+application rather than rely on the current device timezones (Android) or the default `<java.home>/lib`
+version (JVM only).
+
+## Usage
+
+Current version (from [IANA](https://www.iana.org/time-zones)): `2020a`
+
+### Android
+
+Simply add the `dev.zacsweers.ticktock:ticktock-android-tzdb-startup:<version>` dependency. This
+will automatically initialize it appropriately without any configuration needed using `androidx.startup`.
+
+Note that Android usage assumes use of [core library desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring). 
+If you are not using it and/or are minSdk 26+, this library is of no use to you!
+
+### JVM
+
+Add the runtime and tzdb dependencies:
+
+```gradle
+implementation 'dev.zacsweers.ticktock:ticktock-runtime:<version>'
+implementation 'dev.zacsweers.ticktock:ticktock-tzdb:<version>'
```

Comment:
When we add this I think we should name it `ticktock-jvm-tzdb` to match the Android naming.

Reply:
good point, will rename 👍

Reply:
Done b92d109

Reply:
You missed the line the commend is on 😆